assets/js/ab-tests.js

```diff
@@ -0,0 +1,10 @@
+// Test example
```

Comment:
Assuming I could just uncomment this and it would work as a test, I'm a fan of leaving this in. It shows where the tests live, and doesn't send me off on a goose chase if I'm not familiar with the A/B testing setup. It might benefit from a link to those docs for further reading though.

Reply:
it's the "I can uncomment this and code that _not our code right now will run_" part that I'm apprehensive about =)

At the very least, let's change this commented code so it has no _real_ effect even when uncommented.

Reply:
I'm just going to link to the docs if that's all the same. I agree with Alan that having an example there is a nice convenience, but if it goes against other principles, having an example that has no effect isn't as helpful. I would rather just a link at that point.

Also, once we have had a test history, I dunno if any of this matters as much.

Reply:
I'm adding some more info to our readme and docs.